lib/modules/filteReddit.js

```diff
@@ -586,6 +586,23 @@ addModule('filteReddit', function(module, moduleID) {
 						return thing.classList.contains('over18');
 					}
 				},
+				loggedInAs: {
+					name: 'When logged in as a certain user',
+					defaultTemplate: function(patt) {
+						return {type: 'loggedInAs', patt: patt || ''};
+					},
+					fields: [
+						'logged in as /u/', {type: 'text', id: 'loggedInAs', validator: RegExp}
+					],
+					evaluate: function(thing, data, config) {
+						var myName = RESUtils.loggedInUser();
+						if (!myName) {
+							return false;
+						}
+						var pattern = new RegExp('^('+data.loggedInAs+')$', 'i');
```

Comment:
@gamefreak, is there a reason why you used regular expressions instead of string comparisons? In my opinion, this should rather depend on the regexpFilters setting.